lambda/src/index.js

```diff
@@ -1,11 +1,13 @@
 const fetch = require('node-fetch')
-const fs = require('fs')
+const fs = require('fs/promises')
```

Comment:
Do we need to include this in package.json? because I can't seem to run this file locally anymore because it can't find this module?

Reply:
It's [built into Node](https://nodejs.org/api/fs.html#fs_fs_promises_api).

>I can't seem to run this file locally anymore

That's concerning. The Node documentation doesn't have any metadata saying what Node version it was added in. 🤔

Reply:
Is it still broken for you if we change that to `require(fs).promises`?

Reply:
No that actually works

Reply:
@GaryGSC should I change it to `require('fs').promises` then?

Reply:
Yeah.

Reply:
Done